flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/StreamTask.java

```diff
@@ -396,12 +397,13 @@ protected StreamTask(
 
         environment.getMetricGroup().getIOMetricGroup().setEnableBusyTime(true);
         this.throughputCalculator = environment.getThroughputCalculator();
-        this.bufferDebloatPeriod = getTaskConfiguration().get(BUFFER_DEBLOAT_PERIOD).toMillis();
+        Configuration taskManagerConf = environment.getTaskManagerInfo().getConfiguration();
```

Comment:
Can you create a separate jira ticket for this bug fix (as this bug is already on the master)? This fix can stay in this PR, but can you adjust it's commit message to match the crated ticket?

Reply:
done. https://issues.apache.org/jira/browse/FLINK-23726